src/storage/src/hummock/local_version_manager.rs

```diff
@@ -12,63 +12,128 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::borrow::Borrow;
 use std::collections::btree_map::BTreeMap;
-use std::ops::DerefMut;
+use std::ops::{DerefMut, RangeBounds};
 use std::sync::{Arc, Weak};
 use std::time::Duration;
 
-use itertools::Itertools;
 use parking_lot::{Mutex, RwLock};
-use risingwave_pb::hummock::{HummockVersion, Level};
+use risingwave_common::config::StorageConfig;
+use risingwave_pb::hummock::{HummockVersion, Level, SstableInfo};
 use risingwave_rpc_client::HummockMetaClient;
 use tokio::sync::mpsc::error::TryRecvError;
 use tokio::sync::mpsc::{UnboundedReceiver, UnboundedSender};
 use tokio_retry::strategy::jitter;
 
+use super::shared_buffer::shared_buffer_batch::SharedBufferBatch;
+use super::SstableStoreRef;
 use crate::hummock::shared_buffer::shared_buffer_manager::SharedBufferManager;
 use crate::hummock::utils::validate_table_key_range;
 use crate::hummock::{
     HummockEpoch, HummockError, HummockResult, HummockVersionId, INVALID_VERSION_ID,
 };
+use crate::monitor::StateStoreMetrics;
 
-#[derive(Debug)]
-pub struct ScopedLocalVersion {
-    version: Arc<HummockVersion>,
-    unpin_worker_tx: UnboundedSender<Arc<HummockVersion>>,
+/// Ssts that have been uploaded to S3 by the local node but not committed.
+#[derive(Debug, Clone)]
+pub struct UncommittedSsts {
+    pub ssts: BTreeMap<HummockEpoch, Vec<SstableInfo>>,
 }
```

Comment:
Seems introducing struct `UncommittedSsts` is not that necessary. Impl field and functions directly in `LocalVersion`?